CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ Bugfixes:
 Fixes a bug where the transitive closure of a module's dependencies
 where included in the `depends` field in the output for `purs graph`.
 
+* Add white outline stroke to logo in README (#4003, @ptrfrncsmrph)
```

Comment:
Can you move this into the “other” section please? I think this section ought to be for bug fixes inside the actual compiler executable.

Reply:
Done